chatkit-core/src/main/kotlin/com/pusher/chatkit/CurrentUser.kt

```diff
@@ -233,45 +359,104 @@ class CurrentUser(
             callback: (Result<Int, Error>) -> Unit
     ) = makeCallback({ syncCurrentUser.sendMessage(roomId, messageText, attachment) }, callback)
 
+    /**
+     * Send a [messageText] to the [roomId]. The [callback] will be called with a [Result] when the
+     * operation is complete. If the operation was successful you will receive an [Int] which
+     * contains the message id, or an [Error] informing you of what went wrong.
+     */
     fun sendSimpleMessage(
             roomId: String,
             messageText: String,
             callback: (Result<Int, Error>) -> Unit
     ) = makeCallback({ syncCurrentUser.sendSimpleMessage(roomId, messageText) }, callback)
 
+    /**
+     * @see [sendSimpleMessage]
+     */
     fun sendSimpleMessage(
             room: Room,
             messageText: String,
             callback: (Result<Int, Error>) -> Unit
     ) = makeCallback({ syncCurrentUser.sendSimpleMessage(room, messageText) }, callback)
 
+    /**
+     * Sends some message [parts] to the [room]. For more information on what the [parts] can be check
+     * out the [docs][https://pusher.com/docs/chatkit/reference/android].
+     * The [callback] will be called with a [Result] when the operation is complete. If the
+     * operation was successful you will receive an [Int] which contains the sent message id, or an
+     * [Error] informing you of what went wrong.
+     *
+     * e.g.
+     * `sendMultipartMessage(room, listOf(`
+     * `NewPart.Inline("hello world"),`
+     * `NewPart.Url("https://example.com/uploads/myphoto.png", "image/png") ), callback = { })`
```

Comment:
Is there a way to format this better for display (e.g. with indentation)? Perhaps `<code>` or `<pre>`, or if this is actually markdown, a 4 space indent before the lines?

Reply:
I've created a ticket for formatting in leankit as i'm not sure it displays markdown as we'd expect in many places.

Reply:
:+1:

Reply:
Do we need to complete that ticket before merging this to master then?

Reply:
:+1: